src/microservices/Microservices.IsIdentifiable/Reporting/Failure.cs

```diff
@@ -62,25 +62,6 @@ public bool HaveSameProblem(Failure other)
                 string.Equals(ProblemField, other.ProblemField, StringComparison.CurrentCultureIgnoreCase);
         }
 
-        public bool HasOverlappingParts(bool includeExactRangeMatches)
```

Comment:
There a reason to delete this?

Reply:
It's unused except in a couple of tests, though I didn't mean to commit that removal here. I've re-added it now.

Interestingly, the OverlappingMatches_ExactOverlap test now fails since the Equality method for FailurePart is now implemented correctly. Previously it was missing the `Classification` field, so two FailureParts with different classifications were being evaluated as equal.

I think the fix to apply here should be:

```diff
       public void OverlappingMatches_ExactOverlap(bool includeExact)
        {
            Failure f = new Failure(new[]
                {
-                    new FailurePart("Freq",FailureClassification.Person,0),
+                    new FailurePart("Freq",FailureClassification.Organization,0),
                    new FailurePart("Freq",FailureClassification.Organization,0),
                })
```

Though I'm not familiar with this part of the codebase, so may be wrong.

Reply:
I think that was intentional.  2 Failures are the same if they have the same origin text and parts being identified within.

Reply:
What should the fix for this PR be then? Is it worth changing the logic and keeping those tests even though the code is unused (as far as the VS Find can tell me).

Reply:
Lets revert the changes to Failure.  Theres a lot of collections and sets in reviewer/isidentifiable so we don't want to break anything we can always change it over to Equ later